src/AccountTile/doc/AccountTile.mdx

```diff
@@ -0,0 +1,247 @@
+        userThumbnail: {
+          url: null,
+          letters: 'JH'
+        },
+        orgThumbnail: {
+          url: null,
+          letters: 'E'
+        }
+      }
+
+      return (
+        <AccountTile
+          actions={placeholders.actions}
+          user={placeholders.user}
+          isAuthenticated={placeholders.isAuthenticated}
+          orgName={placeholders.orgName}
+          userThumbnail={placeholders.userThumbnail}
+          orgThumbnail={placeholders.orgThumbnail}
+        />
+      );
+    }
+
+    return <AccountTileExample />;
+
+}}
+
+</Playground>
+
+## Multiple Accounts
+
+<Playground style={{ background: "#f3f3f3" }}>
+  {() => {
+    const AccountTilesExample = () => {
+      
+      const accounts = [
+        { 
+          user: {
+            username: 'jharig',
+            fullName: 'Joey Harig',
+            firstName: 'Joey',
+            lastName: 'Harig'
+          },
+          isAuthenticated: true,
+          orgName: 'Esri Proffesional Services',
+          userThumbnail: null,
+          userThumbnail: {
+            url: null,
+            letters: 'JH'
+          },
+          orgThumbnail: {
+            url: null,
+            letters: 'E'
+          }
+        },
+        { 
+          user: {
+            username: 'selse',
+            fullName: 'Someone Else',
+            firstName: 'Someone',
+            lastName: 'Else'
+          },
+          isAuthenticated: false,
+          orgName: 'Esri',
+          userThumbnail: {
+            url: null,
+            letters: 'SE'
+          },
+          orgThumbnail: {
+            url: null,
+            letters: 'E'
+          }
+        }
+      ];
+
+      const createActions = (user, isAuthenticated) => {
+        let actions = [{
+            label: 'Remove account', onClick: () => console.log('remove')
+        }];
+        if (isAuthenticated) {
+          actions.unshift({
+            label: 'Sign out', onClick: () => console.log('sign out')
+          });
+        } else {
+          actions.unshift({
+            label: 'Sign in', onClick: () => console.log('sign in')
+          });
+        }
+        return actions;
+      };
+
+      const accountTiles = accounts.map(account => {
+        return <AccountTile
+          key={account.user.fullName}
+          actions={createActions(account.user, account.isAuthenticated)}
+          user={account.user}
+          isAuthenticated={account.isAuthenticated}
+          orgName={account.orgName}
+          userThumbnail={account.userThumbnail}
+          orgThumbnail={account.orgThumbnail}
+        />
+      });
+
+      return <Panel>{accountTiles}</Panel>;
+    }
+
+    return (
+      <AccountTilesExample />
+    );
+
+}}
+
+</Playground>
+
+## With the `useAccountManager` hook
+
+<Playground style={{ background: "#f3f3f3" }}>
+  {() => {
+    const UseAccountManagerExample = () => {
+      
+      const options = {
+        clientId: "I2YAIyLxcjzok07Y",
```

Comment:
2 thoughts here:

#### First

We need to add `*.netlify.app` to the list of registered redirect URIs for this item so that the sample is functional.

#### Second

We should make sure this item is put in a safe place so that it doesn't accidentally get deleted and break this sample.

Should we make @codylawson the owner? Whether we do that or not it should be [delete protected](https://www.esri.com/arcgis-blog/products/arcgis-online/mapping/prevent-accidental-deletion/).

Reply:
Updated with clientId from an app registered by @codylawson

Reply:
Updated in AccountTile.mdx and in ArcgisAccount.mdx